packages/nuxt/src/pages/runtime/page.ts

```diff
@@ -14,6 +14,23 @@ import { LayoutMetaSymbol, PageRouteSymbol } from '#app/components/injections'
 // @ts-expect-error virtual file
 import { appKeepalive as defaultKeepaliveConfig, appPageTransition as defaultPageTransition } from '#build/nuxt.config.mjs'
 
+export interface NuxtPageProps extends RouterViewProps {
+  /**
+   * Define global transitions for all pages rendered with the `NuxtPage` component.
+   */
+  transition?: boolean | TransitionProps
+
+  /**
+   * Control state preservation of pages rendered with the `NuxtPage` component.
+   */
+  keepalive?: boolean | KeepAliveProps
+
+  /**
+   * Control when the `NuxtPage` component is re-rendered.
+   */
+  pageKey?: string | ((route: RouteLocationNormalizedLoaded) => string)
+}
```

Comment:
It seems we can’t export `NuxtPageProps` like `NuxtLink`. Any suggestions on how we could handle this?

Reply:
We could re-export, as we do here:

- https://github.com/nuxt/nuxt/blob/f209158352b09d1986aa320e29ff36353b91c358/packages/nuxt/src/app/types.ts#L1
- https://github.com/nuxt/nuxt/blob/33789bec056b6409bd99cdf520db53d7bee62076/packages/nuxt/src/app/index.ts#L11

Reply:
Thank you for your guidance.